Automattic/jetpack
# Search: Improve contrast on links in dark mode

This PR makes a small change to the color we use on links in dark mode, bringing the color contrast on the black background from 4.34:1 to 5.63:1 (we want over 4.5:1 [to get the minimum accessibility AA rating](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast))

Fixes #25137

#### Changes proposed in this Pull Request:

* updates the active link color on dark mode from color studio Blue 50 to Blue 40

#### Other information:



#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* On a site with Jetpack Instant Search enabled, choose dark mode from the settings: `/wp-admin/admin.php?page=jetpack-search-configure`.
* Check that the active link color for the sorting links is the color `#1689db`  ![image](https://user-images.githubusercontent.com/30754158/182085630-3b160cbc-9fd5-4120-8d46-000938ef8ee6.png)
* Inspect any link with your browser accessibility tools. Check that the following notice is **no longer** displayed:  <img width="347" alt="Screen Shot 2022-07-20 at 15 43 28" src="https://user-images.githubusercontent.com/17325/179892004-2695888c-6b50-4be4-b4e2-0d52046c4bee.png">
* note: you will still see a contrast notice. It will just be for grey on black instead of blue on black. This is as expected, and is referencing the contrast of the text behind the overlay, which we have faded out on purpose. 
* It is also possible to check that the contrast ratio between two colors is over 4.5:1 using this tool: https://webaim.org/resources/contrastchecker/ the background color is `#000000` and the new link text color is `#1689db`
